operations/mimir-tests/test-consul-multi-zone-generated.yaml

```diff
@@ -2363,14 +2368,18 @@ apiVersion: apps/v1
 kind: StatefulSet
 metadata:
   annotations:
+    grafana.com/prepare-downscale-http-path: store-gateway/prepare-shutdown
+    grafana.com/prepare-downscale-http-port: "80"
+    grafana.com/rollout-downscale-leader: store-gateway-zone-a
     rollout-max-unavailable: "50"
   labels:
+    grafana.com/min-time-between-zones-downscale: 15m
+    grafana.com/prepare-downscale: "true"
     rollout-group: store-gateway
   name: store-gateway-zone-b
   namespace: default
 spec:
   podManagementPolicy: Parallel
-  replicas: 1
```

Comment:
I don't understand. Why this change?

Reply:
This is done on `StatefulSet`s of `zone-b` and `zone-c`, since their number of replicas will be handled by the rollout operator, while the `StatefulSet` of `zone-a` is handled by kubernetes.

During the migration we also need to flux-ignore the stateful sets before the change is merged.

Reply:
Ok, but the change is unrelated from your PR: Why do we see such diff now?